stack/logging.rst

```diff
@@ -30,35 +41,47 @@ Logger names use ``.`` as component separators, not ``::``, even in C++.
 Basic Usage in Python
 =====================
 
-The basic Python interface of :lmod:`lsst.log` is made to be somewhat similar to Python logging.
 The simplest way to log is:
 
 .. code-block:: python
 
-   import lsst.log
-   lsst.log.info('Some information during normal operation')
-   lsst.log.warn('Here is a warning!')
+   import logging
+   logging.info('Some information during normal operation')
+   logging.warning('Here is a warning!')
 
 The example logs to the default (root) logger.
+By default Python logs warning messages and no other log messages.
+This means that the above example code will only write a single log message.
+Application code is required to configure the logging state for libraries and this is generally done by calling :func:`logging.basicConfig` to set a default logger level and default log format.
```

Comment:
But we do by default configure to produce INFO messages, so only people writing non-Task code would observe this.